action/README.md

```diff
@@ -9,19 +9,32 @@ This is a wrapper for [chromatic-cli](https://github.com/chromaui/chromatic-cli)
 In your git repository, create a file `.github/workflows/chromatic.yml` with the following contents:
 
 ```yml
-name: 'Chromatic'
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: 'Chromatic Deployment'
+
+# Event for the workflow
 on: push
 
+# List of jobs
 jobs:
-  test:
+  chromatic-deployment:
+    # Operating System
     runs-on: ubuntu-latest
+    # Job steps
     steps:
       - uses: actions/checkout@v1
-      - run: yarn
-      - uses: chromaui/action@v1
+      - name: Install dependencies
+        run: yarn
+        # 👇 Adds Chromatic as a step in the workflow
+      - name: Deploy to Chromatic
```

Comment:
Shouldn't it be "Publish to Chromatic" since this is typically the phrasing we use?